docs/source/setup/depend.rst

```diff
@@ -110,8 +110,10 @@ FORCE can also be installed on other Liunx distributions (e.g. CentOS). The inst
      cd ~/src/opencv
      wget https://github.com/opencv/opencv/archive/4.1.0.zip
      unzip 4.1.0.zip
+     cd opencv-4.1.0
      mkdir build
      cd build
      cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=/usr/local ..
      make -j
      sudo make install
+     sudo make clean
```

Comment:
Is sudo really necessary here?

Reply:
A clean after installing OpenCV is highly desirable in my opinion. Without cleaning after install, the Docker image was beyond 6 GB (without it around 1.8 GB ). Here I only added for helping the documentation. Sudo I guess is needed if it was needed for the make install command.

Reply:
Agreed, cleaning makes sense. But sudo wasn't used for the build, only for the install ;) Cleaning the temporary files in the build directory should work without sudo.